hardware/raspberrypi/bootmodes/bootflow_2711.md

```diff
@@ -19,12 +19,12 @@ The bootflow for the ROM (first stage) is as follows:-
       * Success - run `recovery.bin` and update the SPI EEPROM or switch to USB mass storage device mode
       * Fail - retry USB device boot
 
-N.B. Currently only CM4 reserves a GPIO for `nRPIBOOT`
+N.B. Currently only CM4 reserves a GPIO for `nRPIBOOT`.
 
 ## recovery.bin
 `recovery.bin` is a minimal second stage program used to reflash the bootloader SPI EEPROM image.
 
-# Second stage bootloader 
+## Second stage bootloader 
```

Comment:
Perhaps there ought to be a `## First stage bootloader ` header too? :shrug:

Reply:
OK - I've added one.